KerbalStuff/blueprints/api.py

```diff
@@ -611,7 +611,8 @@ def create_mod():
     mod.default_version_id = version.id
     db.commit()
     set_game_info(Game.query.get(game))
-    notify_ckan.delay(mod.id, 'create')
+    if ckan:
+        notify_ckan.delay(mod.id, 'create')
```

Comment:
Should this be calling `send_to_ckan` instead (to generate a pull request)? I think `notify_ckan` just asks the NetKAN web hook to inflate/index an existing mod, and wouldn't have any downstream effects if no `.netkan` file has been created yet.

Reply:
Looks like it. And for mod creations the web frontend doesn't seem to use the API from what I saw now.